README.md

```diff
@@ -2,11 +2,16 @@
 Felles applikasjon for opplasting av dokumenter via S3 i SBS. 
 Brukes ifm vedlegg i søknadsdialoger og generering av pdf fra html for brev 
 
-## Kjøre lokalt for søknad
+## Kjøre lokalt for søknad for BA
```

Comment:
I stedet for å fokusere på BA og EF så kunne man skrevet. Kjør med lokal cookie og kjør med fakedings

Reply:
Enig, rettet.